### ISSUE TITLE
Create Years of Potential Life Lost

### ISSUE THREAD

**Issue by lhendey** [NeighborhoodInfoDC/Vital#27]
There are a few ways to do this. See for example: https://www.cdc.gov/mmwr/preview/mmwrhtml/00001773.htm 
 http://www.countyhealthrankings.org/explore-health-rankings/what-and-why-we-rank/health-outcomes/mortality/premature-death/premature-death-ypll 
Here’s an article referencing YPLL and DC: https://www.citizen.org/our-work/health-and-safety/tale-three-cities-racial-and-ethnic-disparities-premature-2 

https://www.cdc.gov/injury/wisqars/fatal_help/definitions_ypll.html

**Comment by maggiesu0725** [comment]
So the basic logic is simple as we talked: YPLL is defined as the number of years of life lost by persons before reaching the cutoff age of interest. These differences are then added for all deceased people in that category.
There are some variations in how we could present these aggregated numbers
1.  Simple total years of potential life lost 
2.  Rate of YPLL per 100,000 population (this is what the public citizen DC table used )
3. Age adjust to standard year when there's multiple year in the calculation
4. Compare the YPLL for different categories, most popular ones are disease type since it is designed to measure death prevention in public health. So I think for our equity analysis, we should exclude accidental death from the calculation, because that's not something that's preventable, but maybe we should.

once we make decisions about the cutoff age and which categories we want to compare, the calculation should not be too complicated. Please let me know if I left out anything. @lhendey

**Comment by lhendey** [comment]
@maggiesu0725 - should we go with 75 as cut-off age? what is county health ranking using?

I think we'd want to do both 1 & 2. 
On 3 - i'm not quite following what this is. can you give an example?
4. if it's not hard we might want to go ahead and do this for a few common categories. i don't think we should exclude accidental deaths from the overall calculation - many of them are preventable - sometimes through public education - (like properly installing car seats).

**Comment by lhendey** [comment]
@maggiesu0725 @ptatian - i looked at this a little bit more. the age-adjusting is done to allow for comparisons across geographies  - the age structure of a population from one place to the next might look different, without age adjusting the differences in YPLL might be do to differences in underlying age structure rather than premature mortality. 

In terms of the equity tool - we'd end up with an indicator: XXXX years of potential life lost per 100,000 (or maybe 1,000 since we are looking at Neighborhoods?)  (According to County Health Rankings DC's rate is 7,700 years lost per 100,000.  I don't think we can turn it into a percentage to fit the framework of the others. But it is still a rate, so maybe he can work with that? 

it seems to me the steps to calculate are:
- [ ] (i think we can start with the current deaths_2016 file rather than editing it) create age and age by race/ethnicity categories (don't need race now -but might as well). 
- [ ] summarize by age categories and geography (neighborhood, ward, city) (repeat for age by race)
- [ ] create weights from census 2010 (Peter - what do you think? not ideal but give us the most detail?) for age groups and age by race and denominator for rates. (weights  = number in group/total number of people in geographic unit)
- [ ] summarize population weights by geography 
- [ ] merge population and deaths together by geography
- [ ] Calculate YPLL, applying age group weights using method in https://urbanorg.app.box.com/file/313701186603 (but use 75 instead of 65).

**Comment by lhendey** [comment]
@maggiesu0725 @ptatian - this one may be easier to digest. http://core.apheo.ca/index.php?pid=190 
(from Laura McKieran in San Antonio. )

**Comment by ptatian** [comment]
Thanks, @lhendey. I'm just not sure about the rate in terms of what the equity statement looks like. While visual displays rates, the equity statement has to count something that gets you to equity. What are we counting? 

Yes, I would not do rates per 100,000 as that's too large a scale for neighborhoods. 

Are the weights for the age-adjusting? Why would you adjust for race, too? cc: @maggiesu0725

**Comment by lhendey** [comment]
@ptatian @maggiesu0725 

yes you are right - weights are for the age-adjusting - shouldn't be done by race. I'll edit the above comment. 

I think the equity statement still works? 

For example DC's overall YPLL per 100,000 K is 7,700 and the black rate is 12,500 to get the "equity" gap you would solve for the "number of years of life gained" if we could make the black rate equal to 7,700. (That would be based on the actual population of blacks.) So say we assume there are 350K blacks in the city to get a rate of 7,700 per 100,000 we would need only 26950 years of lives lost instead of the 43750 years that produced the 12,500 rate. That means we have a potential gain of 16,800 years of life for blacks if we had an equitable city.

**Comment by lhendey** [comment]
@ptatian yes. that is one critique of the indicator- difficult to understand.  having trends would be useful (not for this). 

in the advantages in the box link above "reduction in years of potential life lost is an important public health goal because it reflects the reduction in premature death. Deaths at younger ages are more likely to be attributable to preventable causes and therefore subject to prevention and intervention. In addition, a reduction in premature death is also desirable from a social or economic standpoint."

**Comment by lhendey** [comment]
@maggiesu0725 @ptatian - apparently standard practice for age-adjusted mortality rates is to use the "standard population" which is the age structure in 2000. This webpage gives you instructions on doing so. https://ibis.health.state.nm.us/resource/AARate.html  

Yipeng can you take a look and begin coding this? - we still want to do the years of life lost as well. I think we are going to need to do rolling three year-rates since we are doing this for such small geographies. 

Peter - for the denominators - since we are doing rolling three year averages - census 2010 seems easiest?

**Comment by maggiesu0725** [comment]
@lhendey @ptatian just to confirm, we want to do both age adjusted mortality and years of life lost?

**Comment by lhendey** [comment]
@maggiesu0725 yes. also can you do two sets of weights for the age-adjusted rate? one that uses the "standard pop" and one that uses DC's age structure as the standard?

**Comment by maggiesu0725** [comment]
@lhendey this program is done, please let me know if you want to look at the data together
